dev/com.ibm.ws.ejbcontainer.timer.persistent_fat/fat/src/com/ibm/ws/ejbcontainer/timer/persistent/fat/tests/PersistentTimerCoreTest.java

```diff
@@ -185,4 +220,214 @@ private static void setLateTimerThresholdConfiguration(Long lateTimerThreshold)
+        // Default when no failover is ALL
+        testMissedTimerAction(null, false);
+    }
+
+    /**
+     * Test Persistent Timer missed action default behavior when failover has been enabled.
+     * The default behavior with failover should be ONCE. <p>
+     *
+     * This test will confirm the following :
+     * <ol>
+     * <li> Interval timer will skip expirations missed because of a delay.
+     * <li> Timer.getNextTimeout() will return values in the future; skipping missed expirations.
+     * </ol>
+     */
+    @Test
+    public void testMissedTimerActionDefaultWithFailover() throws Exception {
+        // Default when failover is enabled is ONCE
+        testMissedTimerAction(null, true);
+    }
+
+    /**
+     * Test Persistent Timer missed action "ALL" behavior when failover has not been enabled. <p>
+     *
+     * This test will confirm the following :
+     * <ol>
+     * <li> Interval timer will run all expirations despite a delay that causes several to be missed.
+     * <li> Timer.getNextTimeout() will return values in the past for missed expirations.
+     * </ol>
+     */
+    @Test
+    @Mode(Mode.TestMode.FULL)
+    public void testMissedTimerActionAllNoFailover() throws Exception {
+        testMissedTimerAction("ALL", false);
+    }
+
+    /**
+     * Test Persistent Timer missed action "ALL" behavior when failover has been enabled. <p>
+     *
+     * This test will confirm the following :
+     * <ol>
+     * <li> Interval timer will run all expirations despite a delay that causes several to be missed.
+     * <li> Timer.getNextTimeout() will return values in the past for missed expirations.
+     * </ol>
+     */
+    @Test
+    public void testMissedTimerActionAllWithFailover() throws Exception {
+        testMissedTimerAction("ALL", true);
+    }
+
+    /**
+     * Test Persistent Timer missed action "ONCE" behavior when failover has not been enabled. <p>
+     *
+     * This test will confirm the following :
+     * <ol>
+     * <li> Interval timer will skip expirations missed because of a delay.
+     * <li> Timer.getNextTimeout() will return values in the future; skipping missed expirations.
+     * </ol>
+     */
+    @Test
+    public void testMissedTimerActionOnceNoFailover() throws Exception {
+        testMissedTimerAction("ONCE", false);
+    }
+
+    /**
+     * Test Persistent Timer missed action "ONCE" behavior when failover has been enabled. <p>
+     *
+     * This test will confirm the following :
+     * <ol>
+     * <li> Interval timer will skip expirations missed because of a delay.
+     * <li> Timer.getNextTimeout() will return values in the future; skipping missed expirations.
+     * </ol>
+     */
+    @Test
+    @Mode(Mode.TestMode.FULL)
+    public void testMissedTimerActionOnceWithFailover() throws Exception {
+        testMissedTimerAction("ONCE", true);
+    }
+
+    /**
+     * Test Persistent Timer missed action "ONCE" behavior when failover has not been enabled. <p>
```

Comment:
Copy/paste error -- comment should read "NONE"

Reply:
Will correct with next test update that will add server restart variations.